### PR TITLE
feat(core-flows,dashboard): adds item validations for claims, returns and exchanges

### PR DESCRIPTION
**Pull request by riqwan**
what:

- adds item validations for claims, returns and exchanges
- prevents autofocus on item add
- reset item quantity when error

### CODE REVIEW

File: packages/core/core-flows/src/order/workflows/claim/confirm-claim-request.ts [hunk 1]
```diff
@@ -52,12 +55,36 @@ export const confirmClaimRequestValidationStep = createStep(
     orderClaim: OrderClaimDTO
     orderChange: OrderChangeDTO
   }) {
+    console.log("orderChange -- ", orderChange)
```
Comment (fPolic) [inline packages/core/core-flows/src/order/workflows/claim/confirm-claim-request.ts]: nit: rm

**Review by fPolic — APPROVED** [review]
LGTM!

File: packages/core/core-flows/src/order/workflows/exchange/confirm-exchange-request.ts [hunk 2]
```diff
@@ -58,6 +61,29 @@ export const confirmExchangeRequestValidationStep = createStep(
...
+const confirmIfExchangeItemsArePresent = createStep(
+  "confirm-if-exchange-items-are-present",
+  async function ({
+    exchangeItems,
+    returnItems,
+  }: {
+    exchangeItems: OrderExchangeItemDTO[]
+    returnItems?: OrderReturnItemDTO[]
+  }) {
+    if (exchangeItems.length > 0 || (returnItems || []).length > 0) {
```
Comment (riqwan) [inline packages/core/core-flows/src/order/workflows/exchange/confirm-exchange-request.ts]: q: should an exchange have both inbound and outbound items?

Comment (riqwan) [inline packages/core/core-flows/src/order/workflows/exchange/confirm-exchange-request.ts]:
```suggestion
    if (exchangeItems.length > 0 && (returnItems || []).length > 0) {
```

File: packages/core/core-flows/src/order/workflows/claim/confirm-claim-request.ts (hunk 1 above)
Comment (riqwan) [inline packages/core/core-flows/src/order/workflows/claim/confirm-claim-request.ts]:
```suggestion
```

File: packages/core/core-flows/src/order/workflows/exchange/confirm-exchange-request.ts (hunk 2 above)
Comment (fPolic) [reply packages/core/core-flows/src/order/workflows/exchange/confirm-exchange-request.ts]: I would say yes 👍

**Pull request merged**